### PR TITLE
Fix MessageInputBar translucent property

### PR DESCRIPTION
**Pull request by zhongwuzw**
1. Change `backgroundView.backgroundColor` from `.white` to `color.withAlphaComponent(1.0)` because user can change `backgroundColor`.
2. Add `MessageInputBar` `backgroundColor` to support `translucent` feature .

### CODE REVIEW

File: Sources/Views/MessageInputBar.swift [hunk 1]
```diff
@@ -74,7 +74,10 @@ open class MessageInputBar: UIView {
             }
             blurView.isHidden = !isTranslucent
             let color: UIColor = backgroundView.backgroundColor ?? .white
-            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : .white
+            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : color.withAlphaComponent(1.0)
```
Comment (SD10) [inline Sources/Views/MessageInputBar.swift]: 👍

Comment (nathantannar4) [reply Sources/Views/MessageInputBar.swift]: 👍

**Review by nathantannar4 — APPROVED** [review]

File: Sources/Views/MessageInputBar.swift [hunk 2]
```diff
@@ -74,7 +74,10 @@ open class MessageInputBar: UIView {
...
+            
+            let bgColor: UIColor = backgroundColor ?? .inputBarGray
+            backgroundColor = isTranslucent ? bgColor.withAlphaComponent(0.75) : bgColor.withAlphaComponent(1.0)
         }
```
Comment (nathantannar4) [inline Sources/Views/MessageInputBar.swift]: We can't have this. If you run this you will see it breaks the translucent functionality. We already have:
```swift
/// Also sets the backgroundView's backgroundColor to the newValue
    open override var backgroundColor: UIColor? {
        didSet {
            backgroundView.backgroundColor = backgroundColor
        }
    }
```

**Review by nathantannar4 — CHANGES_REQUESTED** [review]

File: Sources/Views/MessageInputBar.swift (hunk 1 above)
Comment (nathantannar4) [reply Sources/Views/MessageInputBar.swift]: Noticed some older inconsistencies, can you actually change this to:
```
let color: UIColor = backgroundView.backgroundColor ?? .inputBarGray
            backgroundColor = .clear
            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : color
```
I wanted to make it such that the topStackView can have a different background color which is why we only want to adjust the backgroundView's color

Comment (zhongwuzw) [reply Sources/Views/MessageInputBar.swift]: @nathantannar4 ,In your opinion , backgroundView's backgroundColor is not equal to MessageInputBar's backgroundColor, we should delete the property observer of backgroundColor?

```
    /// Delete the property observer below ???
    /// Also sets the backgroundView's backgroundColor to the newValue
    open override var backgroundColor: UIColor? {
        didSet {
            backgroundView.backgroundColor = backgroundColor
        }
    }
```

Comment (zhongwuzw) [reply Sources/Views/MessageInputBar.swift]: @nathantannar4 , as your suggested code, default MessageInputBar's backgroundColor is .inputBarGray, when first set translucent to true, then set to false, MessageInputBar's backgroundColor would always set to .clear, it violate the original setting.

```
let color: UIColor = backgroundView.backgroundColor ?? .inputBarGray
            backgroundColor = .clear
            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : color
```

**Review by nathantannar4 — APPROVED** [review]

**Comment by nathantannar4** [comment]
Awesome wanna just add a changelog

**Review by SD10 — APPROVED** [review]
Thanks for the great work @zhongwuzw @nathantannar4 💯

**Pull request merged**